lib/shared_print/phase_3_validator.rb

```diff
@@ -5,7 +5,6 @@
 require "services"
```

Comment:
The use of this class was rather time-bound. This likely does not need to be re-implemented for mariadb, but then again it is shared print so not anywhere near front of the line anyways.